pkg/convert/go_test.go

```diff
@@ -0,0 +1,96 @@
+package convert
+
+import (
+	"bytes"
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+var modelMsg = model.Messages{
+	Language: language.English,
+	Messages: []model.Message{
+		{
+			ID:          "1",
+			Message:     "message1",
+			Description: "description1",
+			Fuzzy:       true,
+		},
+		{
+			ID:          "2",
+			Message:     "message2",
+			Description: "description2",
+			Fuzzy:       false,
+		},
+	},
+}
+
+func TestToGo(t *testing.T) {
+	t.Parallel()
+
+	expected := []byte(`
+	{
+		"language":"en",
+		"messages":[
+			{
+				"id":"1",
+				"meaning":"description1",
+				"message":"",
+				"translation":"message1",
+				"fuzzy":true
+			},
+			{
+				"id":"2",
+				"meaning":"description2",
+				"message":"",
+				"translation":"message2"
+			}
+		]
+	}`)
+
+	buffer := new(bytes.Buffer)
+
+	assert.NoError(t, json.Compact(buffer, expected))
```

Comment:
```suggestion
	if !assert.NoError(t, json.Compact(buffer, expected)){
		return
	}
```
That way if there is an error in `json.compact` test will not continue (it is pointless to continue as everything else will likely fail, the final assertion for sure), so in a result, if this is the reason why the test is failing, it will output only 1 failed assertion instead of x failed assertions. In that way, it pinpoints the exact place where everything has gone wrong.

Reply:
Thank u!